ui/components/ui/mascot/mascot.stories.js

```diff
@@ -8,7 +8,7 @@ const animationEventEmitter = new EventEmitter();
 
 const containerStyle = {
   height: '600px',
-  width: '357px',
+  width: '400px',
```

Comment:
Updating other instances of `357px` that aren't part of the popup update. This one is in a Storybook story.